kuma/settings/common.py

```diff
@@ -1187,9 +1188,13 @@ def pipeline_one_scss(slug, **kwargs):
 SESSION_COOKIE_HTTPONLY = True
 
 # bug 856061
-ALLOWED_HOSTS = config('ALLOWED_HOSTS',
-                       default='developer-local.allizom.org, mdn-local.mozillademos.org',
-                       cast=Csv())
+ALLOWED_HOSTS = config(
+    'ALLOWED_HOSTS',
+    default='developer-local.allizom.org, mdn-local.mozillademos.org',
+    cast=Csv()
+)
+
+LEGACY_DOMAINS = config('LEGACY_DOMAINS', default='', cast=Csv())
```

Comment:
I think the naming should be `LEGACY_HOSTS` for better readibility.

Reply:
Thanks @safwanrahman, I like that better as well.